### PR TITLE
Add missing dependencies

### PR DESCRIPTION
**Pull request by nimrodshn**
added a missing dependency to the Developer Setup instructions (i.e sudo dnf install redhat-rpm-config ).

### CODE REVIEW

**Comment by Fryguy** [comment]
Very surprised this is needed.  @simaishi @carbonin @jrafanie?

File: developer_setup.md [hunk 1]
```diff
@@ -22,6 +22,7 @@
   sudo dnf -y install sqlite-devel                       # For sqlite3 Gem
   sudo dnf -y install nodejs                             # For ExecJS Gem and bower
   sudo dnf -y install libcurl-devel                      # For Curb
+  sudo dnf install redhat-rpm-config
```
Comment (sergio-ocon) [inline developer_setup.md]: sudo dnf -y ?

**Comment by carbonin** [comment]
I'm also surprised by this. @nimrodshn Can you explain why this package is required and what was failing without it?

**Comment by carbonin** [comment]
@nimrodshn An abbreviated version of why this is needed should also be added in a comment following the install line (like the others around it).

**Comment by jrafanie** [comment]
> @nimrodshn An abbreviated version of why this is needed should also be added in a comment following the install line (like the others around it).

@nimrodshn also, can you please provide more information in the commit message.  Why is this missing dependency needed?

3 months from now `Adding missing dependencies` won't be very helpful when looking at git log.

**Comment by nimrodshn** [comment]
I had issues with no4vir (I think this is the how its spelled, correct me if I'm mistaken) when going through the dev-install - after googling a bit i stumbled uppon a link which unfourtenatly I cant find now, where they offered the line above which solved it for me.
I can try and run it again on a vm and see if I can get to a simillar place. But, if you guys dont think its an issue the we can close this PR..

**Comment by jrafanie** [comment]
> I had issues with no4vir (I think this is the how its spelled, correct me if I'm mistaken) when going through the dev-install - after googling a bit i stumbled uppon a link which unfourtenatly I cant find now, where they offered the line above which solved it for me.
> I can try and run it again on a vm and see if I can get to a simillar place. But, if you guys dont think its an issue the we can close this PR..

I have no idea what that is or what's depending on it.  Let's find out that before we merge.

**Comment by nimrodshn** [comment]
I will run the installation again on a vm and see where I end up..

**Comment by jrafanie** [comment]
Thank you @nimrodshn!

**Pull request closed without merging**